lib/house.rb

```diff
@@ -0,0 +1,33 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price
```

Comment:
Keep a close eye on the interaction pattern, we wanted the price to be returned as an integer; so you would want to do some manipulation to the price argument here.